pages/common/surge.md

```diff
@@ -0,0 +1,19 @@
+# Surge
+
+> Simple, single-command web publishing.
+
+- Upload a new site:
+
+`surge {{ /path/to/my-project }}`
+
+- Deploy site to custom domain (note DNS records must be changed on domain):
```

Comment:
Perhaps `note that the DNS records must be changed on the domain` would read better?

Reply:
Haha - updated the wording in the most recent change 😄